virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-menu/launchbar-menu.component.ts

```diff
@@ -47,6 +50,8 @@ export class LaunchbarMenuComponent implements MVDHosting.LoginActionInterface{
   propertyWindowPluginDef : DesktopPluginDefinitionImpl;
   public authenticationManager : MVDHosting.AuthenticationManagerInterface;
   public appFilter:string="";
+  public activeIndex:number;  
+  private isContextMenuPresent:boolean = false;
```

Comment:
One small thing, can you set this in the constructor with the other initializations?

Reply:
done